snudown.h

```diff
@@ -0,0 +1,118 @@
+/* snudown.h - header for snudown */
+
+/*
+* Copyright (c) 2017, Evgenii Timofeev
```

Comment:
Nit: We haven't added copyright headers to any of the files added in our fork of Sundown, but c/w should be assigned to Reddit if we do.